network-api/networkapi/wagtailpages/utils.py

```diff
@@ -172,11 +172,16 @@ def get_content_related_by_tag(page, result_count=3):
 
     results = False
     own_tags = page.tags.all()
+    own_locale = getattr(page, 'locale', None)
```

Comment:
According to https://docs.wagtail.io/en/stable/reference/pages/model_reference.html this should be a well defined property, so `page.locale` should work.

Reply:
The pipeline was failing because some page did not have the `locale` property. Let me see if I can find it.

Reply:
There it is: https://github.com/mozilla/foundation.mozilla.org/runs/4873707413?check_suite_focus=true#step:10:662

Reply:
Ha, weird. 

I totally missed https://docs.wagtail.io/en/stable/reference/pages/model_reference.html#wagtail.core.models.Page.locale

I thought is only on `TranslatableMixin`: https://docs.wagtail.io/en/stable/reference/pages/model_reference.html#wagtail.core.models.TranslatableMixin.locale